### ISSUE TITLE
Should BootdevTag header tag be set to TagType::Bootdev ?

### ISSUE THREAD

**Issue by jr-oss** [rust-osdev/multiboot2#243]
I noticed this by coincidence and wondered whether the implementation is correct or a copy-paste error.

In bootdev.rs the header tag is set to `TagType::Apm`
```rust
header: TagHeader::new(TagType::Apm, mem::size_of::<Self>() as u32),
```
https://github.com/rust-osdev/multiboot2/blob/main/multiboot2/src/bootdev.rs#L22

Other tag implementations use `Self::ID`, which would be `TagType::Bootdev` for `BootdevTag`

**Comment by phip1611** [comment]
Great catch! Copy and paste error.. Thanks! Do you want to set up a PR?

**Comment by jr-oss** [comment]
I don't have a fork of multiboot2 and currently no planned project where I would use it.
If you could fix it in your copy, that would be nice.
Thanks,
Ralf

**Comment by phip1611** [comment]
Fixed with [v0.23.1](https://crates.io/crates/multiboot2/0.23.1).
